packages/bus-optimize-query-operation/lib/ActorOptimizeQueryOperation.ts

```diff
@@ -1,5 +1,6 @@
 import type { IAction, IActorArgs, IActorOutput, IActorTest } from '@comunica/core';
 import { Actor } from '@comunica/core';
+import type { ActionContext } from '@comunica/types';
```

Comment:
Could you also add `@comunica/types` as dependency to this package?

Reply:
Done!